### PR TITLE
Move constants to configuration file

### PR DESCRIPTION
**Pull request by kingdish**
Inspired by @CactusPuppy 's comment in #45 , I propose to move some constants to a centralized place instead of placing them at the top of files.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -66,6 +66,18 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
+    config.conference = {
+      events_per_page: 3,
+      default_logo:    'snapcon_logo.png',
```
Comment (cycomachead) [inline config/application.rb]: These things are good, but I think we should make them be environment variables, with a default value. `DEFAULT_LOGO_FILENAME || ...` type deal.

File: config/application.rb [hunk 2]
```diff
@@ -66,6 +66,18 @@ class Application < Rails::Application
...
+      default_color:   '#0B3559'
+    }
+
+    config.mailbot = {
+      bcc_address:    'messages@snap.berkeley.edu',
+      email_template: 'email_template',
```
Comment (cycomachead) [inline config/application.rb]: This, IMO should be just left in the mailbot file. It's unlikely to change without significant changes to app code, so it probably doesn't make sense as a config value.

File: config/application.rb [hunk 3]
```diff
@@ -66,6 +66,18 @@ class Application < Rails::Application
...
     config.active_job.queue_adapter = :delayed_job
 
+    config.conference = {
+      events_per_page: 3,
+      default_logo:    'snapcon_logo.png',
+      default_color:   '#0B3559'
+    }
+
+    config.mailbot = {
+      bcc_address:    'messages@snap.berkeley.edu',
```
Comment (cycomachead) [inline config/application.rb]: This definitely should be an environment variable. `OSEM_MESSAGE_BCC_ADDRESS`, perhaps. :)

**Review by cycomachead — COMMENTED** [review]
A couple minor comments, but I like the way this is going! :)

**Review by CactusPuppy — APPROVED** [review]
LGTM

**Pull request merged**